src/codec/ClientAddClusterViewListenerCodec.ts

```diff
@@ -51,13 +52,18 @@ export class ClientAddClusterViewListenerCodec {
         return clientMessage;
     }
 
-    static handle(clientMessage: ClientMessage, handleMembersViewEvent: (version: number, memberInfos: MemberInfo[]) => void = null, handlePartitionsViewEvent: (version: number, partitions: Array<[UUID, number[]]>) => void = null): void {
+    static handle(
+        clientMessage: ClientMessage,
+        handleMembersViewEvent: (version: number, memberInfos: MemberInfo[], connectionManager: ClientConnectionManager) => void = null,
+        handlePartitionsViewEvent: (version: number, partitions: Array<[UUID, number[]]>) => void = null,
+        connectionManager: ClientConnectionManager
+    ): void {
         const messageType = clientMessage.getMessageType();
         if (messageType === EVENT_MEMBERS_VIEW_MESSAGE_TYPE && handleMembersViewEvent !== null) {
             const initialFrame = clientMessage.nextFrame();
             const version = FixSizedTypesCodec.decodeInt(initialFrame.content, EVENT_MEMBERS_VIEW_VERSION_OFFSET);
             const memberInfos = ListMultiFrameCodec.decode(clientMessage, MemberInfoCodec.decode);
-            handleMembersViewEvent(version, memberInfos);
+            handleMembersViewEvent(version, memberInfos, connectionManager);
```

Comment:
These are for the invoke() method that will be called somewhere inside these methods